client/src/components/Register/Introduction.scss

```diff
@@ -0,0 +1,53 @@
+$feature-list-content-margin: 20px;
+$feature-list-icon-size: 50px;
+$feature-list-action-list-spacing: 45px;
+
+$font-feature-list-title-size: 16px;
+$font-feature-list-title-weight: bold;
+
+.mfa-feature-list-title {
+  color: $color-grey-text;
+  font-weight: $font-feature-list-title-weight;
+  font-size: $font-feature-list-title-size;
+  margin-bottom: $feature-list-content-margin;
+}
+
+.mfa-feature-list {
+  list-style: none;
+  margin: 0 0 $feature-list-action-list-spacing;
+  padding: 0;
+}
+
+.mfa-feature-list-item {
+  color: $color-dark-grey-text;
+  display: flex;
+
+  &__icon {
+    height: $feature-list-icon-size;
+    margin-right: $feature-list-content-margin;
+    width: $feature-list-icon-size;
+  }
+
+  &__title {
+    color: $color-dark-grey-text;
+    line-height: 20px;
```

Comment:
I think we should add a line height variable

Reply:
Good call, added (locally). One line-height will not suit all font sizes or contexts though - maybe we need to consult Jared on what each of them should be.